Cargo.toml

```diff
@@ -8,12 +8,16 @@ authors = [
 ]
 description = "Simple NMEA 0183 parser"
 license = "Apache-2.0"
-keywords = ["NMEA", "gps", "glonass", "coordinate", "position"]
+keywords = ["NMEA", "gps", "glonass", "coordinate", "position", "galileo", "gnss"]
```

Comment:
As I know crates.io can have 5 keywords maximum ? 
https://github.com/rust-lang/cargo/issues/2098?ysclid=l5nsajaqq8277955927#issuecomment-227603969

Reply:
Ahhh.. yes, I completely forgot about that... Will remove them.

Reply:
Fixed. I also added a category.
Can you think of a category addition to crates.io that best fits the NMEA crate? I guess it could be something like `Marine electronics` or just `Marine`